client/data/courses/types.ts

```diff
@@ -0,0 +1,29 @@
+export type CourseSlug = 'blogging-quick-start';
+
+export type VideoSlug =
```

Comment:
Why do we need to explicitly define the video slugs? They might change in the future. If this is added then the change would need to be done in 2 places (backend & frontend)

Reply:
Okay, I changed it to string to make it simple. See: https://github.com/Automattic/wp-calypso/pull/59514